### PR TITLE
Feat/header

### PR DESCRIPTION
**Pull request by artemn0va**
Still to do:
1. Responsive between 768px and 1440px.
2. Position of button "ПРОЕКТЫ"
3. Animation of button "ПОДДЕРЖАТЬ"
4. Add footer to opened hamburger menu

### CODE REVIEW

File: blocks/topnav/__social/topnav__social.css [hunk 1]
```diff
@@ -1,22 +1,5 @@
-.topnav__social {
-  display: flex;
-  flex-flow: row;
-}
```
Comment (artemschirov) [inline blocks/topnav/__social/topnav__social.css:1]: Пустой файл. Будет удалён

Comment (artemschirov) [inline blocks/topnav/__social/topnav__social.css:1]: Empty file. Must be deleted

File: blocks/topnav/__button/topnav__button.css [hunk 2]
```diff
@@ -1,11 +0,0 @@
-.topnav__button {
```
Comment (artemschirov) [inline blocks/topnav/__button/topnav__button.css:1]: Empty file. Must be deleted

File: scripts/hamburger.js [hunk 3]
```diff
@@ -1,17 +1,31 @@
+const body = document.querySelector(".body");
```
Comment (artemn0va) [inline scripts/hamburger.js:1]: "в файлах .js кавычки одинарные, в .css .html - двойные"

File: styles/index.css [hunk 4]
```diff
@@ -14,20 +14,20 @@
...
 @import url(../blocks/topnav/__logo/topnav__logo.css);
 @import url(../blocks/topnav/__menu/topnav__menu.css);
 @import url(../blocks/topnav/__items/topnav__items.css);
-@import url(../blocks/topnav/__item/topnav__item.css);
-@import url(../blocks/topnav/__link/topnav__link.css);
+@import url(../blocks/topnav/__item-link/topnav__item-link.css);
 @import url(../blocks/topnav/__buttons/topnav__buttons.css);
 @import url(../blocks/topnav/__button/topnav__button.css);
 @import url(../blocks/topnav/__socials/topnav__socials.css);
 @import url(../blocks/topnav/__social/topnav__social.css);
```
Comment (artemn0va) [inline styles/index.css:29]: Empty file. Must be deleted

File: styles/index.css [hunk 5]
```diff
@@ -14,20 +14,20 @@
...
 /* topnav */
 @import url(../blocks/topnav/topnav.css);
 @import url(../blocks/topnav/__logo/topnav__logo.css);
 @import url(../blocks/topnav/__menu/topnav__menu.css);
 @import url(../blocks/topnav/__items/topnav__items.css);
-@import url(../blocks/topnav/__item/topnav__item.css);
-@import url(../blocks/topnav/__link/topnav__link.css);
+@import url(../blocks/topnav/__item-link/topnav__item-link.css);
 @import url(../blocks/topnav/__buttons/topnav__buttons.css);
 @import url(../blocks/topnav/__button/topnav__button.css);
```
Comment (artemn0va) [inline styles/index.css:26]: Empty file. Must be deleted

File: blocks/topnav/__social/topnav__social.css (hunk 1 above)
Comment (artemn0va) [inline blocks/topnav/__social/topnav__social.css:1]: Empty file. Must be deleted

**Review by uladzimirfilipau — APPROVED** [review]

**Review by temirm — APPROVED** [review]

**Review by uladzimirfilipau — APPROVED** [review]

**Pull request closed without merging**